src/java.base/share/classes/java/time/LocalDate.java

```diff
@@ -2147,11 +2147,16 @@ public int hashCode() {
      */
     @Override
     public String toString() {
+        var buf = new StringBuilder(10);
+        formatTo(buf);
+        return buf.toString();
+    }
+
+    void formatTo(StringBuilder buf) {
```

Comment:
For methods used outside of a class (i.e. not private) a minimal method description is recommended.